modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -464,6 +468,14 @@ private void stopSchedulers() {
         retryScheduler = null;
       }
     }
+
+    if (reconnectScheduler != null) {
```

Comment:
to create and destroy the scheduler on each use is not a good way of handling the lifecycle